### DISCUSSION TITLE
Security Onion Installation in Graphical mode

### DISCUSSION THREAD

**Discussion by safdar-aneela** [Security-Onion-Solutions/securityonion#1792] [Unsupported Versions] [answered]
Hi, 
I have used security onion for quite a while since it has 14 and 16 versions. I recently have installed sec onion 2.3.2 version which is build on CentOs. I am having difficulty in using graphical interface since after installation I am prompted with a command line interface. Is this how the newer version is or I can install it in graphical too?

**Comment by safdar-aneela** [comment, marked as answer]
As previous versions were not a web based setup, it confused me a little. Now I get that security onion to be accessed now with a browser.

Thanks and its a welcome change.

**Comment by greatapoc** [comment]
There is a graphical interface you can install with the command so-analyst-install. You can run it on the same machine as the install https://docs.securityonion.net/en/latest/analyst-vm.html

**Reply by safdar-aneela** [reply]
Thank you for your response. This script needs one to be online. However I am working on offline system. Is there any other way to run it offline?
